### PR TITLE
allow extension store ID to be specified separately from study ID

### PR DESCRIPTION
**Pull request by rhelmer**
It looks like the store ID in the Chrome URL cannot be changed (although it can on Firefox). 

Currently we map the `studyId` specified in the extension to a hyphenated version (`study-id`), but this is not guaranteed to be the same, so let's make it possible to set this separately.

### CODE REVIEW

File: extensions/attention-stream/src/background.ts [hunk 1]
```diff
@@ -49,6 +49,9 @@ const enableEmulatorMode = Boolean(__ENABLE_EMULATOR_MODE__);
 // The Rally-assigned Study ID.
 let studyId = "attentionStream";
 
+// The "slug" on the extension store.
+let storeId = "rally-attention-stream";
```
Comment (rhelmer) [inline extensions/attention-stream/src/background.ts:53]: @aaga one thing I've noticed is that AMO and CWS somehow generated different slugs from the same `manifest.json` - on CWS it is `rally-attention-stream`, and on AMO it is `attention-stream`.

It looks like we're not allowed to change this on CWS but we are on AMO. We *could* allow multiple slugs here, but I'd like to try to unify them for consistency's sake.

Maybe we should modify AMO and change the URL in Firestore at the same time, wdyt?

Comment (aaga) [reply extensions/attention-stream/src/background.ts:53]: agreed!

**Review by aaga — APPROVED** [review]
Sorry this is so delayed.

I agree about unifying store slugs.

Given that we've had different slugs, does that mean this functionality was broken on CWS?

Also, how does our RWP deployment work these days? Would we need to land this, change AMO, and update firebase all at the same time?

**Comment by rhelmer** [comment]
> Sorry this is so delayed.
> 
> I agree about unifying store slugs.
> 
> Given that we've had different slugs, does that mean this functionality was broken on CWS?

Yes, I was testing this (in preparation to actually send it somewhere useful) and noticed that it wasn't matching the Chrome store URL due to the store ID not matching the study ID.

> Also, how does our RWP deployment work these days? Would we need to land this, change AMO, and update firebase all at the same time?

I think the AMO is the only one of of step, so I suspect we just need to push this extension update and then change the AMO slug to match CWS, and be done with it.

**Comment by rhelmer** [comment]
> e out

Oh right and the URL in Firestore pointing to the AMO store of course we'd need to change as well. I don't really like changing URLs like this but I feel like it's better to do it now vs. waiting since we're not linking directly to it anywhere, yet.

**Pull request merged**